common/views/components/CompactCard/CompactCard.js

```diff
@@ -47,11 +48,24 @@ const CompactCard = ({
   StatusIndicator,
   ExtraInfo,
   xOfY,
+  type,
```

Comment:
I don't feel great about using the _actual_ `CompactCard` component for the `MediaObject`. My thinking was the the styles could be shared between both components, but that they'd have different names. It's a bit pedantic, but we have a particular type of thing in mind when we refer to 'cards', and the `MediaOject` doesn't fit with it. 

I thought we were already using `styled-components` in this file, which would mean you could export them and re-use in a new `MediaObject` component file. Looks like were still using sass here, though. You could either continue using the sass in both component files, or re-write the styles as `styled-components` (only if you and other reviewers agree, that is).

Reply:
Completely agree with you. 
The current implementation is using the old style convention of sass. I am happy to implement a version of compact-card which allows style-components to be added. 

@jamesgorrie  @jamesgorrie  @davidpmccormick My question is shall I refactor this now to make compact card typescript and allow to add styled components as this is also used in a number of places within the application. It may change the scope. 

Happy todo either way!

Reply:
My preference would be to do it the 'right' way first time, instead of creating a bit of tech debt. I don't think it would be much more expensive to do it that way in terms of time.

Reply:
Sounds like no objections.. will convert this to typescrip,  update the styles and make sure any reference to compact card is working the same, whilst im at it write some tests for compact cards.

Reply:
That feels like a nicer approach @davidpmccormick